assets/dev/scss/frontend/widgets/icon-box.scss

```diff
@@ -127,18 +100,6 @@
 
 	@media (max-width: $screen-mobile-max ) {
 		@include icon-position(-mobile);
-
-		&.elementor-position- {
-
-			&left,
-			&right {
-
-				.elementor-icon-box-icon {
-					display: block;
-					flex: unset;
-				}
-			}
-		}
```

Comment:
This removes unused `.elementor-position-{right|left}` css class from the Icon Box widget.